frontend/src/components/molecules/Navbar/Navbar.tsx

```diff
@@ -18,14 +19,21 @@ const Navbar = () => {
   };
 
   return (
-    <Flex h="7.5vh" w="100%" bg="blue" justifyContent="flex-end">
+    <Flex h="7.5vh" w="100%" bg="darkgreen" justifyContent="flex-end">
       <Center mr=".75rem">
         {authenticated ? (
           <Button onClick={logout}> Logout </Button>
         ) : (
           <Button onClick={moveToLogin}> Login </Button>
         )}
       </Center>
+      <Center mr=".75rem">
+        {authenticated ? (
```

Comment:
conditional render is more suitable then ternary

Reply:
I will try to move this, hold up :smiley_cat:

Reply:
I fixed it

Reply:
:100: